Libs/MRML/Core/vtkMRMLVolumeArchetypeStorageNode.cxx

```diff
@@ -789,8 +819,7 @@ std::string vtkMRMLVolumeArchetypeStorageNode::UpdateFileList(vtkMRMLNode *refNo
   // test whether refNode is a valid node to hold a volume
   if (!refNode->IsA("vtkMRMLScalarVolumeNode") )
   {
-    vtkErrorToMessageCollectionMacro(this->GetUserMessages(), "vtkMRMLVolumeArchetypeStorageNode::UpdateFileList",
-      "Reference node is not a vtkMRMLVolumeNode");
+    vtkErrorMacro("vtkMRMLVolumeArchetypeStorageNode::UpdateFileList: Reference node is not a volume");
```

Comment:
This is a developer-level error so I guess you decided not to report it to the user, but shouldn't the user get some indication of the issue?

Reply:
The user will get a high-level error message that reading of the image failed and more details will be in the log.

I decided not to display these because it would be unnecessary burden for translators to translate these low-level messages that are really only meant for developers, they don't tell anything useful to users. It is also easier for developers if log messages are kept in English.

Reply:
Yes, that's what I figured, just wanted to be sure